snippets/card-collection.liquid

```diff
@@ -28,6 +29,8 @@
     assign card_color_scheme = settings.collection_card_color_scheme
     assign card_style = settings.collection_card_style
   endif
+  assign desktop = columns_desktop | at_least: 1
+  assign mobile = columns_mobile | at_least: 1
```

Comment:
Same here for the naming. Having something a bit more descriptive so it's easier to read would be great.